styles.css

```diff
@@ -0,0 +1,69 @@
+body {
+  min-width: 375px;
+  margin: 0;
+  padding: 0;
+
+  /* box-sizing: border-box; */
+}
+
+.menu {
+  width: 1.25rem;
+  margin-left: 17.75rem;
+  margin-top: 1.188rem;
+}
+
+header {
+  background-image: url(./icons/image_geometry_1.png);
+  background-repeat: no-repeat;
+  height: 4.625rem;
+  display: flex;
+  align-items: center;
+}
+
+.homepage {
+  width: 20rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+}
```

Comment:
- [ ] Kindly consider avoiding the use of fixed `width` or `height` for your page to ensure responsiveness. Instead, consider responsive sticking to the default behavior of your page, and style your page content accordingly to allow the page content to adjust dynamically based on the user's screen size. This approach will help create a more adaptable and mobile-friendly layout, ensuring that your page is responsive across various devices and screen resolutions. When your page is set on a responsive screen the Chrome dev tool below is what it looks like, Kindly make sure to fix it as it would help create a good user experience.

<table>
<tr>
<th align=“middle”>Figma design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="318" alt="Screenshot 2023-07-05 at 19 35 23" src="https://github.com/JohnteFresh/IrunguJohnson-Mobile-Portfolio/assets/107192076/f27b0f40-8e9e-44dd-9678-aae6b38910d7">
</td>
<td>
<img width="345" alt="Screenshot 2023-07-05 at 19 34 31" src="https://github.com/JohnteFresh/IrunguJohnson-Mobile-Portfolio/assets/107192076/f6b36fa2-d295-4bd3-894b-f88b39de6031">
</td>
</tr>
</table>